pkg/services/pluginsintegration/clientmiddleware/metrics_middleware.go

```diff
@@ -26,21 +29,26 @@ type pluginMetrics struct {
 type MetricsMiddleware struct {
 	pluginMetrics
 	pluginRegistry registry.Service
+	features       featuremgmt.FeatureToggles
 	next           plugins.Client
 }
 
-func newMetricsMiddleware(promRegisterer prometheus.Registerer, pluginRegistry registry.Service) *MetricsMiddleware {
+func newMetricsMiddleware(promRegisterer prometheus.Registerer, pluginRegistry registry.Service, features featuremgmt.FeatureToggles) *MetricsMiddleware {
+	var additionalLabels []string
+	if features.IsEnabled(featuremgmt.FlagPluginsInstrumentationStatusSource) {
+		additionalLabels = []string{"status_source"}
```

Comment:
@scottlepp the PR adds the `status_source` label to those metrics:

- `plugin_request_total`
- `plugin_request_duration_milliseconds`
- `plugin_request_duration_seconds`

As @marefr pointed out, just adding the new label to `plugin_request_duration_seconds` may be enough.

Do you have any thoughts on this, based on your use case/alerts that you want to define?

Reply:
Seems a bit confusing having source only on `plugin_request_duration_seconds`, but i also understand that it's redundant to have it on all of these.  I believe @bergquist suggested using `plugin_request_duration_seconds` so maybe he can chime in here.

Reply:
No strong opinions by me.  As far as for SLO we can have it on just one.  Just seemed a bit weird.